arch/arm/src/armv7-m/arm_busfault.c

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
+ * arch/arm/src/armv7-m/arm_busfault.c
```

Comment:
let's update armv6-m too

Reply:
ARMv6-M uses HardFault for all fault conditions